web/src/app/new-account/page.tsx

```diff
@@ -2,7 +2,16 @@ import CompanyLogo from "../components/company-logo";
 import FormTitle from "../components/form-title";
 import NewAccountForm from "./components/newAccountForm";
 
-export default function NewAccount() {
+async function getLevels() {
```

Comment:
```suggestion
async function getLevels(): Promise<Array<string>> {
```

Só tipar a resposta pra evitar qualquer erro no compilador